src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -227,15 +228,17 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
         this.currentStep = E2EIHandlerStep.INITIALIZED;
         void this.enroll();
       },
-      secondaryActionFn: () => {
-        this.showE2EINotificationMessage();
+      secondaryActionFn: async () => {
+        const isFreshMLSSelfClient = await this.coreE2EIService.isFreshMLSSelfClient();
+
+        this.showE2EINotificationMessage(isFreshMLSSelfClient);
```

Comment:
Let's save this info in the `config` (since it's passed to `initialize`) and avoid recomputing it (or do we need to recompute it at this point?)